### PR TITLE
Feature/textloader without dask

### PR DESCRIPTION
**Pull request by julesbertrand**
## Description

Follow up of #154 to solve #153

## Related Issue

#153

## Type of Change

- [ ] 📚 Examples / docs / tutorials / dependencies update
- [ ] 🔧 Bug fix (non-breaking change which fixes an issue)
- [x] 🥂 Improvement (non-breaking change which improves an existing feature)
- [ ] 🚀 New feature (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🔐 Security fix

## Checklist

- [x] I've read the [`CODE_OF_CONDUCT.md`](https://github.com/artefactory/NLPretext}/blob/main/CODE_OF_CONDUCT.md) document.
- [x] I've read the [`CONTRIBUTING.md`](https://github.com/artefactory/NLPretext}/blob/main/CONTRIBUTING.md) guide.
- [x] I've updated the code style using `make codestyle`.
- [x] I've written tests for all new methods and classes that I created.
- [x] I've written the docstring in Google format for all the methods and classes that I used.

### CODE REVIEW

File: nlpretext/_utils/daskloader.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+def read_json(files_path, encoding):
+    return dd.read_json(files_path, encoding=encoding)
+
+
+def read_csv(files_path, encoding):
+    return dd.read_csv(files_path, encoding=encoding)
+
+
+def read_parquet(files_path, encoding):
+    return dd.read_parquet(files_path, encoding=encoding)
```
Comment (sachalasry-artefact) [inline nlpretext/_utils/daskloader.py:19]: Even though it's already clear, type hinting and a small docstring would be nice I think

File: nlpretext/textloader.py [hunk 2]
```diff
@@ -104,16 +142,21 @@ def _read_text_parquet(self, files_path):
...
 
     def read_text(
-        self, files_path, file_format=None, encoding=None, compute_to_pandas=True, preprocessor=None
+        self,
+        files_path,
+        file_format=None,
+        encoding=None,
+        compute_to_pandas=True,
+        preprocessor=None,
     ):
```
Comment (sachalasry-artefact) [inline nlpretext/textloader.py:160]: Typehinting on return value

File: tests/test_textloader.py [hunk 3]
```diff
@@ -59,18 +61,39 @@ def test__read_text_txt(mock_read_text):
...
+    dummy_instance = TextLoader(
+        file_format=file_format,
+        use_dask=False,
+        encoding=encoding,
+        text_column=text_column,
+    )
+    dummy_instance._read_text_txt(files_path)
+
+    # Then
+    mock_read.assert_called_once_with(files_path, encoding=encoding)
```
Comment (sachalasry-artefact) [inline tests/test_textloader.py:81]: Result of the function is not tested, is it on purpose?

Comment (julesbertrand) [reply tests/test_textloader.py:81]: Nope, fixing it

File: nlpretext/_utils/daskloader.py (hunk 1 above)
Comment (julesbertrand) [reply nlpretext/_utils/daskloader.py:19]: Ok for type hints, but i tried to add docstrings and it only makes the code more cumbersome

File: nlpretext/textloader.py (hunk 2 above)
Comment (julesbertrand) [reply nlpretext/textloader.py:160]: Done but dask type hints are not streamlined -> complicated with mypy

**Review by sachalasry-artefact — APPROVED** [review]

**Pull request merged**